node/grpc/server.go

```diff
@@ -189,7 +189,7 @@ func (s *Server) RetrieveChunks(ctx context.Context, in *pb.RetrieveChunksReques
 		return nil, err
 	}
 
-	encodedBlobSize := core.GetBlobSize(blobHeader.QuorumInfos[in.GetQuorumId()].EncodedBlobLength)
+	encodedBlobSize := core.GetBlobLength(core.GetEncodedBlobLength(blobHeader.Length, blobHeader.QuorumInfos[in.GetQuorumId()].QuorumThreshold, blobHeader.QuorumInfos[in.GetQuorumId()].AdversaryThreshold))
```

Comment:
shouldn't it still be `GetBlobSize`? `GetBlobLength` takes blob size in bytes but `GetEncodedBlobLength` returns in number of symbols?

Reply:
Good catch!